src/components/Header.tsx

```diff
@@ -10,9 +10,20 @@ export function Header(): JSX.Element {
         </HeaderLink>
 
         <nav>
-          <HeaderLink to="/">Tracks</HeaderLink>
-          <HeaderLink to="/exercises">Exercises</HeaderLink>
-          <HeaderLink to="/stories">Stories</HeaderLink>
+          <ul
+            className="list-unstyled"
+            style={{ display: 'flex', marginBottom: 0 }}
```

Comment:
I think there is `d-flex` and `mb-0` as class names, _but I might be wrong_.

Reply:
There are: see https://getbootstrap.com/docs/4.5/utilities/flex/ and https://getbootstrap.com/docs/4.5/utilities/spacing/
@JesseSingleton Would you mind using the proposed class names?

Reply:
Thanks for the suggestion @SleeplessByte , didn't know they existed. 

@ErikSchierboom Pushed the requested change!

Reply:
(I guessed correctly)